server/views/formPages/incident/report-has-been-deleted.html

```diff
@@ -0,0 +1,20 @@
+{% extends "../../partials/layout.html" %} 
```

Comment:
This page is needed in case user selects the browser back button from report-cancelled page.